src/User/ProfilePage.jsx

```diff
@@ -1,11 +1,45 @@
-import React from "react";
-import {
-  StyledCardHeading,
-  StyledForm,
-  StyledCard
-} from "../StyledFormComponents";
-
-const ProfilePage = ({ user, isLoading, error }) => {
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import { StyledCardHeading } from "../StyledFormComponents";
+
+const StyledCard = styled.ul`
```

Comment:
Don't you already have a styled card you could reuse?

Reply:
We have a reusable one for the add/edit conference and the conference details, but for the profile page I've changed the size dimensions so I don't think I can reuse it. I'll try changing the name of this card though so they're not the same!

Reply:
Stlyed components are supposed to be pretty flexible, you could always pass props and set the size of the card that way instead of creating a new component with the exact same styling